src/main/java/make_comment/interface_adapter/makeCommentController.java

```diff
@@ -0,0 +1,23 @@
+package make_comment.interface_adapter;
+
+import make_comment.use_case.MakeCommentInputBoundary;
+import make_comment.use_case.MakeCommentRequestModel;
+
+public class makeCommentController {
+    private final MakeCommentInputBoundary interactor;
+
+    public makeCommentController(MakeCommentInputBoundary interactor) {
+        this.interactor = interactor;
+    }
+
+    public void passToInteractor(String body, int postId){
+        int commentId = interactor.getNumCommentCreated();
```

Comment:
Unsure about the clean architecture here, especially concerning the single responsibility principle. However, the code seems to work so this is a nonissue

Reply:
do you mean only the body and postId should be included in MakeCommentRequestModel? and commentId and CurrentUserId should be grabbed in the interactor?